README.md

```diff
@@ -82,12 +82,19 @@ The following config options are supported:
 | GITSIGN_OIDC_REDIRECT_URL   | ✅                 |                                  | OIDC Redirect URL                                                                                                                                                                                                                          |
 | GITSIGN_REKOR_URL           | ✅                 | https://rekor.sigstore.dev       | Address of Rekor server                                                                                                                                                                                                                    |
 | GITSIGN_TIMESTAMP_AUTHORITY | ✅                 |                                  | Optional address of timestamping authority. If set, a trusted timestamp will be included in the signature.                                                                                                                                 |
+| GITSIGN_FULCIO_ROOT         | ✅                 |                                  | Path to PEM encoded certificate for Fulcio CA                                                                                                                                                                                              |
 
 For environment variables that support `Sigstore Prefix`, the values may be
 provided with either a `GITSIGN_` or `SIGSTORE_` prefix - e.g.
 `GITSIGN_CONNECTOR_ID` or `SIGSTORE_CONNECTOR_ID`. If both environment variables
 are set, `GITSIGN_` prefix takes priority.
 
+#### Other environment variables
+
+| Environment Variable      | Description                                                                     |
```

Comment:
Do we want to mention `SIGSTORE_FULCIO_ROOT` here too

Reply:
Technically covered above since it's an alias to GITSIGN_FULCIO_ROOT.

SIGSTORE_REKOR_PUBLIC_KEY is special in that it's embedded directly in the cosign library so gitsign doesn't have control over it 😭